### DISCUSSION TITLE
Show infomation about buses arriving to the bus stop

### DISCUSSION THREAD

**Discussion by muralito** [organicmaps/organicmaps#8072] [Q&A]
Hi. In POIs that are public transport stops, in several cities is becoming usual that the stop have a QR code that guides to a mobile site that gives you information about the next buses that are arriving.

I have found a post in OSM Community with this issue: https://community.openstreetmap.org/t/rfc-departures-url-on-e-g-highway-bus-stop-for-online-departures-boards/107673/13

E.g., in Montevideo, for a bus stop ref/id 3228, the URL showing the arriving buses is https://m.montevideo.gub.uy/comoir/parada?numero=3228 (the parameter is the ref/id of the bus stop)

Could this link be shown in the POI information so the user can follow it?
Have any preference in how it should be tagged in OSM?

BTW, in this case of Montevideo, this info is also available from the local government as an API at no cost, and it could be used to display the info in OM itself, but i think that it is much harder to implement  worldwide, as not every city has the same API, and also the authorization token should be managed for each API. Or maybe a mixed solution, in which some cities OM could use the API, and in others follow the link.

**Comment by hb0nd** [comment]
UK/tfl bus stops also have this, tagged as `naptan:AtcoCode'

Eg. 
https://www.openstreetmap.org/node/469770550
is:
 https://tfl.gov.uk/bus/stop/490007515D

(see https://github.com/organicmaps/organicmaps/issues/3730 for related discussion)

**Comment by muralito** [comment]
Yes. In the OSM community discussion that i linked above, Andy Townsend advice was to use tag "website" which is easier to be processed by data consumers (OM in this case).

The question is, is adding a "website" URL that, for several thousands of bus stops, only varies in the parameter (which is also the "ref" or some other tag) a good method to map that info? 

There is a chance that OM in the future uses that "website" tag in bus stop POIs?

It would be better to map the url of the service at the bus network level and only the parameter at each stop? Or OM mao format could handle thousands of URL strings almost equal without impacting in the map size? Or the map size is not relevant at decisions of this kind?

BTW, looking this issue i found another issues related with bus stops.
* They cant be searched. Ej, when you search by text "Parada 4562" i would expect that the following bus stop is found https://www.openstreetmap.org/node/2153319513 (ref=4562)
* Some of them behave different. e.g. it they are tagged with shelter yes, they are not considered as bus stops, and are not searcheable by category. I do not now why, but a closer bus stop (https://www.openstreetmap.org/node/11060782717) is showed but not the 4562.

Below is an annotated screenshot of how this bus stops are displayed in the current version.
![Screenshot_20240503-164947-busstops-annotated](https://github.com/organicmaps/organicmaps/assets/7611868/f0d09bbf-164f-4f90-90b6-97727ea8ae6d)
